modules/test/playwright/tests/portal-web/env/setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e -x
+
+if [ -L $0 ]
+then
+	CURRENT_FILE_NAME=$(readlink $0)
+else
+	CURRENT_FILE_NAME=$0
+fi
+
+CURRENT_DIR_NAME=$(dirname ${CURRENT_FILE_NAME})
+
+source ${CURRENT_DIR_NAME}/../../../env/common.sh
+
+update_portal_ext_properties ${CURRENT_DIR_NAME}/../../../portal-ext.properties
+
+update_portal_ext_properties ${CURRENT_DIR_NAME}/portal-ext.properties
+
+copy_to_deploy_folder ${CURRENT_DIR_NAME}/deploy
+
+deploy_osgi_modules \
```

Comment:
We were thinking in creating this and the `deploy_client_extensions` part as a declarative list in a file (like `modules.list` or something similar) instead of coding it.

I think declarative is a bit better since it is clearer and faster for everyone. At least for very common tasks I find it easier to lookup. And for more intricate setup/teardown we always have the scripts.